From a2a62570dd038a459326e3d56cb593af97e2f6f3 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Thu, 28 Aug 2014 15:44:44 +0100
Subject: Revert "Prefer the GDK windowing backend for Clutter"

This reverts commit 0fc73a07770223964dbf41a20a4b0a8c3ef9a29c.

I could have gotten away with this, if it wasn't for you meddling kids
and your Wayland.

diff --git a/clutter-gtk/gtk-clutter-util.c b/clutter-gtk/gtk-clutter-util.c
index 53ec095..f66f8a9 100644
--- clutter-gtk/gtk-clutter-util.c
+++ clutter-gtk/gtk-clutter-util.c
@@ -66,9 +66,6 @@ gtk_clutter_init_internal (void)
   display = gdk_display_get_default ();
 
 #if defined(CLUTTER_WINDOWING_GDK)
-  /* we want the GDK backend by default */
-  clutter_set_windowing_backend (CLUTTER_WINDOWING_GDK);
-
   if (clutter_check_windowing_backend (CLUTTER_WINDOWING_GDK))
     {
       clutter_gdk_set_display (gdk_display_get_default ());
@@ -82,7 +79,7 @@ gtk_clutter_init_internal (void)
   if (clutter_check_windowing_backend (CLUTTER_WINDOWING_X11) &&
       GDK_IS_X11_DISPLAY (display))
     {
-      /* enable ARGB visuals by default for Clutter; this is a no-op */
+      /* enable ARGB visuals by default for Clutter */
       clutter_x11_set_use_argb_visual (TRUE);
 
       /* share the X11 Display with GTK+ */
@@ -113,9 +110,7 @@ gtk_clutter_init_internal (void)
     }
   else
 #endif
-    {
-      g_error ("*** Unsupported backend.");
-    }
+    g_error ("*** Unsupported backend.");
 
   /* We disable clutter accessibility support in order to not
    * interfere with gtk accessibility support.
-- 
cgit v0.10.1

